.github/actions/rdp/action.yaml

```diff
@@ -0,0 +1,69 @@
+name: Azure RDP setup
+description: 'Connect via RDP in order to log in as session other than 0.'
+
+inputs:
+  az_vm_hostname:
+    description: Hostname for "xfreerdp /v:az_vm_hostname"
+    required: true
+  az_vm_username:
+    description: Username for "xfreerdp /u:az_vm_username"
+    required: true
+  az_vm_password:
+    description: Password for "xfreerdp /p:az_vm_password"
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    - name: rdp
+      shell: bash
+      env:
+        DEBIAN_FRONTEND: noninteractive
+        MAX_CONNECTION_ATTEMPTS: "10"
+      run: |
+        set -eo pipefail
```

Comment:
Hum, you marked the previous comment as resolved (I don’t know, we tend to let the reviewer checking/mark things are resolved), I don’t see a -u here?